src/httpAuthScheme.ts

```diff
@@ -358,30 +362,12 @@ export class Token {
         const authenticator = new Uint8Array(input.buffer.slice(offset, offset + len));
         offset += len;
 
-        return new Token(tokenTypeEntry, payload, authenticator);
-    }
-
-    static async fetch(pt: PrivateToken): Promise<Token> {
-        const issuerUrl = await getIssuerUrl(pt.challenge.issuerName);
-        const client = new Client();
-        const tokReq = await client.createTokenRequest(pt);
-        const tokRes = await tokReq.send(issuerUrl, TokenResponse);
-        const token = await client.finalize(tokRes);
-        return token;
+        return new subClassCtor(tokenTypeEntry, payload, authenticator);
     }
 
     serialize(): Uint8Array {
         return new Uint8Array(
             joinAll([this.payload.serialize().buffer, this.authenticator.buffer]),
         );
     }
-
-    verify(publicKey: CryptoKey): Promise<boolean> {
```

Comment:
given you expect this to be defined
```suggestion
abstract verify(publicKey: CryptoKey): Promise<boolean>
```